src/sentry/dynamic_sampling/rules/utils.py

```diff
@@ -31,6 +31,7 @@ class RuleType(Enum):
     BOOST_LATEST_RELEASES_RULE = "boostLatestRelease"
     IGNORE_HEALTH_CHECKS_RULE = "ignoreHealthChecks"
     BOOST_KEY_TRANSACTIONS_RULE = "boostKeyTransactions"
+    PRIORITISE_BY_TX_NAME_RULE = "prioritiseByTxName"
```

Comment:
shall we make this plural to be consistent?

```suggestion
    PRIORITIZE_BY_TRANSACTION_NAMES_RULE = "prioritizeByTransactionNames"
```

Reply:
yeah sure we can rename, I had a similar name but then decided to make it shorter.